asb-ballerina/Ballerina.lock

```diff
@@ -0,0 +1,4 @@
+org_name = "ballerinax"
```

Comment:
Do we need to commit Ballerna.lock file?

Reply:
Added to Gitignore file. Thank you